vendor-extra/LiberoPatternsBundle/src/Resources/views/content-header.html.twig

```diff
@@ -0,0 +1,40 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-header', 'wrapper']) }) %}
+
+<header {%- include '@LiberoPatterns/attributes.html.twig' %}>
+
+  {%- block body -%}
+
+    {%- with contentTitle only -%}
+
+      {%- block title -%}
+
+        {% set length = include('@LiberoPatterns/heading.html.twig')|striptags|trim|length %}
+
+        {% if length < 20 %}
+          {% set lengthClass = 'xx-short' %}
+        {% elseif length < 36 %}
+          {% set lengthClass = 'x-short' %}
+        {% elseif length < 46 %}
+          {% set lengthClass = 'short' %}
+        {% elseif length < 57 %}
+          {% set lengthClass = 'medium' %}
+        {% elseif length < 80 %}
+          {% set lengthClass = 'long' %}
+        {% elseif length < 120 %}
+          {% set lengthClass = 'x-long' %}
+        {% else %}
+          {% set lengthClass = 'xx-long' %}
+        {% endif %}
+
+        {% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-header__title', 'content-header__title--' ~ lengthClass]) }) %}
```

Comment:
the length of this makes it difficult to see the various levels (the second `merge` involving only the `class` attribute); not sure if it will become an helper or if multiple lines can be used to simply reformat it